quattor/types/aquilon/hardware.pan

```diff
@@ -0,0 +1,43 @@
+# This template extends the base schema with Aquilon-provied resources
+
+declaration template quattor/types/aquilon/hardware;
+
+include 'pan/types';
+include 'quattor/functions/hardware';
+include 'quattor/types/annotation';
+include 'quattor/types/sensors';
+include 'quattor/physdevices';
+
+@documentation{
+    Rack definition
+}
+type structure_rack = {
+    "column" : string
+    "name"   : string
+    "room"   ? string
+    "row"    : string
+};
+
+@documentation{
+    system location definition
+}
+type structure_sysloc = {
+    "building"   : string
+    "campus"     ? string
+    "city"       : string
+    "country"    : string
+    "continent"  : string
+    "room"       : string
+    "bunker"     ? string
+    "region"     ? string
+    "dns_search_domains" ? string[]
```

Comment:
I don't remember seeing this in the structure_sysloc I sent you? This wouldn't really belong here I think? Nor location.